rbx_binary/src/text_deserializer.rs

```diff
@@ -10,6 +10,8 @@ use byteorder::{LittleEndian, ReadBytesExt};
 use rbx_dom_weak::types::{Color3, UDim, UDim2, Vector2};
 use serde::{Deserialize, Serialize};
 
+use rbx_dom_weak::types::Axes;
+
```

Comment:
We can merge these imports.

```suggestion
use rbx_dom_weak::types::{Axes, Color3, UDim, UDim2, Vector2};
use serde::{Deserialize, Serialize};

```